conn2res/iodata.py

```diff
@@ -132,10 +139,9 @@ def fetch_dataset(task, n_trials=100, *args, **kwargs):
     if task in NEUROGYM_TASKS:
 
         # create a Dataset object from NeuroGym
-        kwargs = {'dt': 100}

```

Comment:
I removed this line and dt is passed as kwargs when calling fetch_dataset. This way other environmental variables of the ngym.Dataset can be also set (e.g. timing, cohs etc.).

Of note, dt=100 by default for several tasks including PerceptualDecisionMaking, PerceptualDecisionMakingDelayResponse, ContextDecisionMaking, so we might not even want to specify this in the examples anyway.